### DISCUSSION TITLE
Migrating 8 > 10: encoding issue with p:autoComplete

### DISCUSSION THREAD

**Discussion by luccotta** [primefaces/community#1398] [PrimeFaces] [answered]
Hello, 

after migrating from 8 to 10, the `@FacesConverter` I use for my `p:autoComplete` is receiving a value with encoding issues. 

In `getAsString` I return "Português" but `getAsObject` is receiving "PortuguÃªs" when I select it in the` p:autoComplete`

Any idea why? 

I'm using Tomcat 7, Java 11, Chrome

Thanks

**Comment by tandraschko** [comment, marked as answer]
Try PrimeFaces 13 TBH

**Reply by melloware** [reply]
@luccotta yep try PF 13.0.7 i am almost positive this was fixed long ago...

**Reply by luccotta** [reply]
Hi, just tried PF 13.0.8 and the problem still occurs.

**Reply by melloware** [reply]
@luccotta then I need a PF Test reproducer to see your issue. I have 70 applications on 13.0.8 with no issues

Please  provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.
